### DISCUSSION TITLE
How can I achieve an internal border effect on windows? (pictures)

### DISCUSSION THREAD

**Discussion by liiyung** [asmvik/yabai#1104] [Q&A] [answered]
Referring to the effect in the attached picture + other screenshots ("magical sunset", "cherry blossom") [here](https://github.com/6gk/polka) – is it currently possible to achieve the internal border effect? 

![68747470733a2f2f692e726564642e69742f626468616a72626a6e686d34312e706e67](https://user-images.githubusercontent.com/75713129/147215188-bea53c6c-9dfb-4104-b86d-747493b3d462.png)

If yes – how would one go about achieving it? Hopefully I can add it in myself if I understand what to do. 
If no – could I make this a feature request? 

Thank you!

**Comment by asmvik** [comment, marked as answer]
Unfortunately this is not really something that is possible to do on macOS, sorry.
